client/src/main/java/io/pravega/client/stream/EventStreamReader.java

```diff
@@ -25,6 +25,9 @@
      * timeout waiting for them to arrive. If none do, an EventRead will be returned with null for
      * {@link EventRead#getEvent()}. (As well as for most other fields)
      *
+     * An EventRead with null for {@link EventRead#getEvent()} is returned when the Reader has read all events up to the
```

Comment:
Please update this Javadoc

Reply:
As discussed, EventRead with null for EventRead.getEvent() is returned when the endStreamCut is reached.